pkg/koordlet/resmanager/cpu_evict.go

```diff
@@ -97,55 +97,130 @@ func (c *CPUEvictor) cpuEvict() {
 	klog.V(5).Info("cpu evict process finished.")
 }
 
-func (c *CPUEvictor) calculateMilliRelease(thresholdConfig *slov1alpha1.ResourceThresholdStrategy, windowSeconds int64) (*metriccache.BECPUResourceMetric, int64) {
+func (c *CPUEvictor) calculateMilliRelease(thresholdConfig *slov1alpha1.ResourceThresholdStrategy, windowSeconds int64) int64 {
 	// Step1: Calculate release resource by BECPUResourceMetric in window
-	avgBECPUQueryResult := c.resmanager.metricCache.GetBECPUResourceMetric(generateQueryParamsAvg(windowSeconds))
-	if !isAvgQueryResultValid(avgBECPUQueryResult, windowSeconds, c.resmanager.collectResUsedIntervalSeconds) {
-		return nil, 0
+	queryparam := generateQueryParamsAvg(windowSeconds)
+	querier, err := c.resmanager.metricCache.Querier(*queryparam.Start, *queryparam.End)
+	if err != nil {
+		klog.Warningf("get query failed, error %v", err)
+		return 0
 	}
+	// BECPUUsage
+	avgBECPUUsage, count := getBECPUMetric(metriccache.BEResouceAllocationUsage, querier, queryparam.Aggregate)
```

Comment:
milli value should be mentioned in name, e.g. avgBECPUMilliUsage
or use CPU Core as the unit